ci/scripts/install_gcs_testbench.sh

```diff
@@ -38,14 +38,14 @@ case "$(uname -s)-$(uname -m)" in
   Darwin-arm64)
     # Workaround for https://github.com/grpc/grpc/issues/28387 .
     # Build grpcio instead of using wheel.
-    # storage-testbench 0.27.0 pins grpcio to 1.46.1.
-    ${PYTHON:-python3} -m pip install --no-binary :all: "grpcio==1.46.1"
+    # storage-testbench 0.32.0 pins grpcio to 1.49.1.
+    ${PYTHON:-python3} -m pip install --no-binary :all: "grpcio==1.49.1"
```

Comment:
Can we remove this workaround?
It seems that grpcio 1.47.0 or later doesn't have this problem: https://github.com/grpc/grpc/issues/28387#issuecomment-1163468600